cmd/src/server.rs

```diff
@@ -497,14 +489,20 @@ impl<ER: RaftEngine> TiKVServer<ER> {
         gc_worker
     }
 
-    fn init_servers(
-        &mut self,
-        gc_worker: &GcWorker<
-            RaftKv<ServerRaftStoreRouter<RocksEngine, ER>>,
-            RaftRouter<RocksEngine, ER>,
-        >,
-    ) -> Arc<ServerConfig> {
+    fn init_servers(&mut self) -> Arc<ServerConfig> {
+        let gc_worker = self.init_gc_worker();
+        let mut ttl_checker = Box::new(LazyWorker::new("ttl-checker"));
+        let ttl_scheduler = ttl_checker.scheduler();
+
         let cfg_controller = self.cfg_controller.as_mut().unwrap();
+        cfg_controller.register(
```

Comment:
Not sure if it's OK to adjust the order. /cc @NingLin-P

Reply:
It is OK, `ConfigController` use `Arc<RwLock<_>>` internally.